extensions/wikia/Wall/WallHelper.class.php

```diff
@@ -315,9 +314,9 @@ private function getCommentsData( $comments ) {
 
 			$items[$i]['avatar'] = $data['avatarSmall'];
 			$items[$i]['user-profile-url'] = $data['userurl'];
-			$user = User::newFromName( $data['author']->getName() );
+			$user = $data['author'];
 
-			if ( $user ) {
+			if ( $user instanceof User ) {
 				$items[$i]['real-name'] = $user->getName();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "real-name" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)